### ISSUE TITLE
Error with redeclare value

### ISSUE THREAD

**Issue by sirenkovladd** [putoutjs/minify#20]
- **Version** (`npm -v`): 10.5.2
- **Node Version** `node -v`: v20.13.1
- **OS** (`uname -a` on Linux): Darwin arm64

Error with redeclare value

```js
import { minify } from "@putout/minify";

const source = `global.resolveSources = function resolveSources() {
  for (let e = 0, n = this.length; e < n; ++e) {
    const n = this[e]();
    if (void 0 !== n) return n;
  }
}`;

eval(source);
const result = minify(source);
console.log(result);
eval(result);
```

```sh
❯ node i.js
global.resolveSources=function a(){for(let e=0,n=this.length;e<n;++e){var n=this[e]();if(n!==void 0)return n}};
undefined:1
global.resolveSources=function a(){for(let e=0,n=this.length;e<n;++e){var n=this[e]();if(n!==void 0)return n}};
                                                                          ^

SyntaxError: Identifier 'n' has already been declared
```

**Comment by coderaiser** [comment]
Just fixed in [v3.19.6](https://github.com/putoutjs/minify/releases/tag/v3.19.6) 🎉

**Comment by sirenkovladd** [comment]
thank you
